apollo-federation/src/query_plan/query_planner.rs

```diff
@@ -1148,7 +1148,8 @@ type User
               userById(id: 1) {
                 ...userFields
                 id
-              }      another_user: userById(id: 2) {
```

Comment:
These were previously incorrect. This top level field needs to have a new line in between.